### PR TITLE
Spelling, grammar, and other improvements

### PR DESCRIPTION
**Pull request by pgaskin**

### CODE REVIEW

**Comment by pgaskin** [comment]
@flyingmutant, @nefthy, this is ready for review.

**Comment by flyingmutant** [comment]
Typo fixes in the code all look good. However, as for the man page -- I am not a native speaker, can you please help with proofreading the text a bit, @nefthy?

File: Doc/cmus.txt [hunk 1]
```diff
@@ -150,47 +151,46 @@ prepend tracks to the play queue (4)
...
 
 @h1 KEYBINDINGS
 
-Here's list of default keybindings.  See *unbind* and *bind* commands in the
-COMMANDS section.
+Here's list of default keybindings. To change them, see the *unbind* and *bind*
+commands in the COMMANDS section.
 
 
 @h2 Common Context
```
Comment (flyingmutant) [inline Doc/cmus.txt:196]: By the way, are these bindings up to date?

**Comment by pgaskin** [comment]
> proofreading the text a bit, @nefthy?

To help, the main changes I made were:

- Rewrapping the text to 80 cols ([this](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) VSCode extension is really useful).
- Making word forms consistent within the config and command sections.
- Fixing missing articles before nouns (mostly just `the`).
- Fixing misused commas (mostly extraneous ones which change the meaning of the sentence, or ones which should have been semicolons or periods).
- Rewriting a few sections/paragraphs which were overly wordy, incorrect, or unclear
  - `LIBRARY VIEW SORTING`: This one was the worst. The style was completely different from the rest, it jumped around, and it left more questions than it answered about some of the sorting behaviour. I needed to check the code for this one.
  - The help text for some of the commands was quite confusing when it talked about exceptional behaviour (e.g. what happens if no arguments are provided) as different possibilities for the command rather than as exceptions separate from the main description.
  - `FILES`: It wasn't clear what each file actually did compared to the others.
- Removing default bindings from individual command help text (they are duplicates of the main default section).
- Making the style consistent, more technical, and slightly more formal.
  - I might have missed a few areas which were almost good enough, but I've fixed the worst occurrences.

File: Doc/cmus.txt (hunk 1 above)
Comment (pgaskin) [reply Doc/cmus.txt:196]: I'll check that. I'll probably also add a note about them being visible in the last view.

File: Doc/cmus.txt [hunk 2]
```diff
@@ -12,33 +12,33 @@ cmus [*options*]
 
 @h1 DESCRIPTION
 
-cmus is a small ncurses based music player.  It supports various output
-methods by output-plugins. cmus has completely configurable keybindings and
-can be controlled from the outside via *cmus-remote*(1).
+cmus is a lightweight ncurses music player. It supports various output methods
+by using dynamically-loaded output plugins. cmus has configurable keybindings
+and can be controlled externally using *cmus-remote*(1).
```
Comment (pgaskin) [inline Doc/cmus.txt:17]: If anyone has better suggestions for this, I'd be happy to hear them. I only fixed the sentence structure and wording of the previous description, which I don't really like to begin with.

File: Doc/cmus.txt (hunk 1 above)
Comment (pgaskin) [reply Doc/cmus.txt:196]: I'll need to update them. I'll merge the ones from `rc` into the documentation, then I'll update `rc` to match the order of the documentation.

Comment (pgaskin) [reply Doc/cmus.txt:196]: Also, I'll need to add some missing commands to the documentation:

<details>

```
$ grep -E '[{] "[a-z_-]+",\s+cmd_' command_mode.c | cut -d\" -f2 | while read cmd; do printf "%-40s %s\n" "${cmd}" "$(grep -Em1 "^${cmd}\\s" Doc/cmus.txt || echo '!!!')"; done
add                                      add [-l] [-p] [-q] [-Q] <file|dir|url|playlist>
bind                                     bind [-f] <context> <key> <command>
browser-up                               browser-up (*backspace*)
cd                                       cd [directory]
clear                                    clear [-l] [-p] [-q]
colorscheme                              colorscheme <name>
echo                                     echo <arg>...
factivate                                factivate <user-defined-filter>...
filter                                   filter <filter-expression>
fset                                     fset <name>=<filter-expression>
help                                     !!!
invert                                   !!!
live-filter                              live-filter <simple-filter-expression|short-filter-expression>
load                                     load [-l] [-p] <playlist>
lqueue                                   lqueue [NUM]
mark                                     mark <filter-expression>
mute                                     !!!
player-next                              player-next (*b*)
player-pause                             player-pause (*c*)
player-pause-playback                    !!!
player-play                              player-play [filename] (*x*)
player-prev                              player-prev (*z*)
player-stop                              player-stop (*v*)
prev-view                                !!!
left-view                                !!!
right-view                               !!!
pl-create                                pl-create <name>
pl-export                                pl-export <filename>
pl-import                                pl-import [filename]
pl-rename                                pl-rename <name>
push                                     push [text]
pwd                                      !!!
raise-vte                                !!!
rand                                     !!!
quit                                     quit [-i] (*q*, *:wq*)
refresh                                  refresh (*^L*)
run                                      run <command>
save                                     save [-e]  [-l] [-L] [-p] [-q]  [file]  (*:w*)
search-b-start                           !!!
search-next                              search-next (*n*)
search-prev                              search-prev (*N*)
search-start                             !!!
seek                                     seek [+-](<num>[mh] | [HH:]MM:SS)
set                                      set <option>=<value>
shell                                    shell <command>
showbind                                 showbind <context> <key>
shuffle                                  shuffle (false)
source                                   source <filename>
toggle                                   toggle <option>
tqueue                                   tqueue [NUM]
unbind                                   unbind [-f] <context> <key>
unmark                                   !!!
update-cache                             update-cache [-f]
version                                  !!!
view                                     view <name or 1-7>
vol                                      vol [+-]NUM[%] [[+-]NUM[%]]
w                                        !!!
win-activate                             win-activate (*enter*)
win-add-l                                win-add-l [-n] (*a*)
win-add-p                                win-add-p [-n] (*y*)
win-add-q                                win-add-q [-n] (*e*)
win-bottom                               win-bottom (*G*, *end*)
win-down                                 win-down [NUM] (*j*, *down*)
win-half-page-down                       win-half-page-down (*^D*)
win-half-page-up                         win-half-page-up (*^U*)
win-mv-after                             win-mv-after (*p*)
win-mv-before                            win-mv-before (*P*)
win-next                                 win-next (*tab*)
win-page-bottom                          !!!
win-page-down                            win-page-down (*^F*, *page_down*)
win-page-middle                          !!!
win-page-top                             !!!
win-page-up                              win-page-up (*^B*, *page_up*)
win-remove                               win-remove (*D*, *delete*)
win-scroll-down                          win-scroll-down (*^E*)
win-scroll-up                            win-scroll-up (*^Y*)
win-sel-cur                              win-sel-cur (*i*)
win-toggle                               win-toggle (*space*)
win-top                                  win-top (*g*, *home*)
win-up                                   win-up [NUM] (*k*, *up*)
win-update                               win-update (*u*)
win-update-cache                         win-update-cache [-f]
wq                                       !!!
```

</details>

Comment (pgaskin) [reply Doc/cmus.txt:196]: There's a conflict between rc and the docs: doc=`i                               win-sel-cur` rc=`i toggle show_hidden`.

And, stuff in rc but not the docs:

```
u win-update
backspace browser-up
space win-activate
^U win-half-page-up
^D win-half-page-down
i win-sel-cur
k win-up
U win-update-cache
```

Comment (pgaskin) [reply Doc/cmus.txt:196]: I'm going to put this in another PR since this one is getting somewhat large.

Comment (pgaskin) [reply Doc/cmus.txt:196]: I've opened #1037 with these changes.

**Comment by flyingmutant** [comment]
@pgaskin I've merged #1037, can you please rebase this one?

**Comment by pgaskin** [comment]
@flyingmutant, done

File: Doc/cmus.txt [hunk 3]
```diff
@@ -331,79 +331,49 @@ u                       win-update
...
-Generally, in the library view cmus uses three levels of sorting: the first
-level would be the artist name, the second one the album and finally the
-actual track.
+Note: Albums which feature various artists (e.g. samplers or compilations) are
+treated specially. If an album artist tag or the ID3v2 *TPE2* frame is set, it
+will be used instead of the real artist name. Otherwise, cmus determines if the
+album is a compilation (if *albumartist* or *artist* are set to *Various
+Artists*, *Various*, *VA*, or *V/A*; or if *compilation* or *partofacompilation*
+are set to a truthy value; or if the ID3v2 *TCMP* frame is set). If so, the
+album is named *<Various Artists>*.
```
Comment (flyingmutant) [inline Doc/cmus.txt:344]: I think you meant `artist`, not `album`.

Comment (flyingmutant) [reply Doc/cmus.txt:344]: The rest LGTM.

Comment (pgaskin) [reply Doc/cmus.txt]: Fixed.

**Comment by flyingmutant** [comment]
Thank you!

**Pull request merged**